cmd/geth/main.go

```diff
@@ -433,6 +433,31 @@ func startNode(ctx *cli.Context, stack *node.Node, backend ethapi.Backend, isCon
 			utils.Fatalf("Failed to start mining: %v", err)
 		}
 	}
+
+	// Spawn a standalone goroutine for status synchronization monitoring,
+	// if full sync is completed in block specimen creation mode set replica config flag
+	if ctx.GlobalBool(utils.ReplicaEnableSpecimenFlag.Name) || ctx.GlobalBool(utils.ReplicaEnableResultFlag.Name) {
+		log.Info("Synchronisation started, historical blocks synced set to 0")
+		go func() {
+			sub := stack.EventMux().Subscribe(downloader.DoneEvent{})
```

Comment:
according to the struct TypeMux documentation, it is deprecated and should be replaced by Feeds.
See:
https://github.com/ethereum/go-ethereum/pull/16706
https://github.com/ethereum/go-ethereum/pull/20312/

Most places in geth still use TypeMux though and it works, so I would leave this up to you if you want to implement this.

Reply:
why not take that up as a ticket?